mimecast/plugin.spec.yaml

```
@@ -284,6 +284,12 @@ types:
       description: The action requested by the user
       type: string
       required: false
+  deletions:
+    deletedRecordIds:
+      title: Mimecast IDs of Deletions
+      description: The IDs of records deleted from the blocked list
+      type: array
```

```suggestion
      type: array
```
`array` is not a type unless it was created in the custom type section. Please look at https://komand.github.io/python/spec.html?highlight=type#base-types and update the type for `deletedRecordIds`.

---

If the ids are strings just update to `"[]string"`

---

this type was removed.